src/test/kotlin/lotto/domain/prize/LottoPrizeTest.kt

```diff
@@ -2,29 +2,21 @@ package lotto.domain.prize
 
 import org.assertj.core.api.Assertions
 import org.junit.jupiter.params.ParameterizedTest
-import org.junit.jupiter.params.provider.Arguments
-import org.junit.jupiter.params.provider.MethodSource
-import java.util.stream.Stream
+import org.junit.jupiter.params.provider.CsvSource
 
 class LottoPrizeTest {
     @ParameterizedTest
```

Comment:
소소한 팁인데요~
```suggestion
        @ParameterizedTest(name = "{0}개 일치하고 보너스 번호 일치여부가 {1}라면 당첨금액은 {2} 이다.")
```

이렇게 사용하면 아래와 같은 결과물을 얻을수 있습니다~!

<img width="553" alt="image" src="https://user-images.githubusercontent.com/13347548/170975963-9f3e0176-4327-47cd-9d36-e5beb35dc1c4.png">

Reply:
훨씬 보기 명확하고 좋네용 ㅎㅎ 감사합니다!
[9d37e0c](https://github.com/next-step/kotlin-lotto/pull/361/commits/9d37e0cc2f9a29745e31721dba5e61f0936413a9) 에 반영했습니다